htmd/parameterization/fftype.py

```diff
@@ -67,51 +94,39 @@ def fftype(mol, rtfFile=None, prmFile=None, method='GAFF2', acCharges=None, tmpD
     if method not in fftypemethods:
         raise ValueError('Invalid method {}. Available methods {}'.format(method, ','.join(fftypemethods)))
 
+    if method == 'CGenFF_2b6' and acCharges:
+        raise ValueError('acCharges')
+
     if netcharge is None:
         netcharge = int(round(np.sum(mol.charge)))
         logger.warning('Using atomic charges from molecule object to calculate net charge')
 
-    prm = names = elements = atomtypes = charges = impropers = masses = None
     if rtfFile and prmFile:
         logger.info('Reading FF parameters from {} and {}'.format(rtfFile, prmFile))
         prm = parmed.charmm.CharmmParameterSet(rtfFile, prmFile)
         names, elements, atomtypes, charges, masses, impropers = readRTF(rtfFile)
-        # addParmedResidue(prm, names, elements, atomtypes, charges, impropers)
+
     else:
         logger.info('Assigning atom types with {}'.format(method))
-        # Find the executables
```

Comment:
I know this code is ugly, and I understand you removing it, but postponing the error to later may not be the best approach

Reply:
`subprocess.call` checks for missing executables.
`antechamber` and `parmchk2` are packaged together, so it is unlike that one will be missing and the other present.
Code simplicity vs. an error message 10 lines earlier?